src/main/java/com/zzpj/dtos/BookDto.java

```diff
@@ -11,6 +11,8 @@
 @JsonPropertyOrder({"name", "author"})
 public class BookDto {
 
+    public Long id;
+
     @NotNull(message = "name cannot be null!")
```

Comment:
to trzeba będzie zmienic na @NotEmpty zamiast @NotNull żeby nie mogło być pustym stringiem